sync/src/tasks/tests_dag.rs

```diff
@@ -37,7 +34,7 @@ async fn sync_block_process(
         let block_chain_service = async_std::task::block_on(
             registry.service_ref::<BlockConnectorService<MockTxPoolService>>(),
         )?;
-        let dag_fork_height = local_node.chain().dag_fork_height()?.unwrap_or(u64::MAX);
+        let dag_fork_height = local_node.dag_fork_number()?;
```

Comment:
Can we remove this dag_fork_height if it's always zero？Is it only used to check if we can skip the accumultorInfo checking?  Correct me if I've missed something.

Reply:
Yes it can. But currently the Halley fork number is 1 and for starcoin on Halley running, I preserved the value for now.

Reply:
Halley network will be destroyed and redeployed when `dag-master` branch has been updated. So for Halley network, it's ok to change the fork number to zero.